src/main/java/com/google/api/gax/testing/MockGrpcService.java

```diff
@@ -44,8 +44,11 @@
   /** Returns all the requests received. */
   public List<GeneratedMessageV3> getRequests();
 
-  /** Sets the responses. */
-  public void setResponses(List<GeneratedMessageV3> responses);
```

Comment:
Actually, could you keep `setResponse` for the moment and mark it `@Deprecated`? That would make sure no more rollbacks are necessary - we can then remove setResponse a little later.